lib/src/bluez_client.dart

```diff
@@ -258,8 +258,7 @@ class BlueZDevice {
     }
 
     return (value as DBusDict).children.map((key, value) => MapEntry(
-        BlueZManufacturerId((key as DBusUint16).value),
-        processValue(value as DBusVariant)));
+        (key as DBusUint16).value, processValue(value as DBusVariant)));
```

Comment:
Why are you removing the `BlueZManufacturerId`? This is intentionally there to make it clearer to the caller the meaning of the value that is being returned.

Reply:
Please re-submit with just the typo fix.